### ISSUE TITLE
[Doc]: Get-AzKeyVaultCertificate → "save as pfx" example broken

### ISSUE THREAD

**Issue by odegroot** [Azure/azure-powershell#18251]
### Description

"Example 2: Get cert and save it as pfx" does not work on my machine.

Specifically, the following command does nothing.

```ps1
# Write to a file
[System.IO.File]::WriteAllBytes("cert.pfx", $secretByte)
```

I was able to fix the problem by using the following command instead ([source](https://stackoverflow.com/questions/31855705/write-bytes-to-a-file-natively-in-powershell)):

```ps1
Set-Content -Path "cert.pfx" -Value $secretByte -AsByteStream
```

Perhaps this would be a nice improvement to the docs?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 45cbf981-0e5a-cf19-bc73-882d61da8001
* Version Independent ID: 63263710-e864-310a-6a42-c618857872b0
* Content: [Get-AzKeyVaultCertificate (Az.KeyVault)](https://docs.microsoft.com/en-us/powershell/module/az.keyvault/get-azkeyvaultcertificate?view=azps-7.5.0)
* Content Source: [src/KeyVault/KeyVault/help/Get-AzKeyVaultCertificate.md](https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault/help/Get-AzKeyVaultCertificate.md)
* Service: **key-vault**
* GitHub Login: @mikefrobbins
* Microsoft Alias: **mirobb**

**Comment by dingmeng-xue** [comment]
@CaptainFanZzz , please look into this issue. Please adjust example and ensure it works on Windows PowerShell and PowerShell 7 both.

**Comment by BethanyZhou** [comment]
Hi @odegroot , thank you for reporting. I have updated the example with a more concise way. Please have a try.

**Comment by odegroot** [comment]
Thank you, but I don't believe this is right. ([link to updated example](https://github.com/Azure/azure-powershell/blob/main/src/KeyVault/KeyVault/help/Get-AzKeyVaultCertificate.md#example-2-get-cert-and-save-it-as-pfx))

When I try this, `$secret` contains a long Base64 string. Simply writing it to a .pfx-file does not yield anything valid. I do need the Base64-decode step.

Note that the example output (`-----BEGIN PRIVATE KEY-----`) is not a pfx either, that's PEM-formatted.

Additionally, for brevity, we can skip the first step.
```ps1
$cert = Get-AzKeyVaultCertificate -VaultName $vaultName -Name $certName
```
All we use from `$cert` is `$cert.Name` which is the same as `$certName` anyway. We can just input `$certName` into `Get-AzKeyVaultSecret` directly.

I have created a PR that resolves these comments: #19003

**Comment by BethanyZhou** [comment]
Thanks for your contribution. I will review your PR.
